src/appgateway.tf

```
@@ -43,8 +43,8 @@ module "app_gw" {
   name                = format("%s-app-gw", local.project)
 
   # SKU
-  sku_name = "WAF_v2"
-  sku_tier = "WAF_v2"
+  sku_name = var.app_gateway_sku_tier
```

```suggestion
  sku_name = var.app_gateway_sku_name
```
probably it's better to use 2 vars name and tier

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_gateway#tier